omniport/core/open_auth/views/application.py

```diff
@@ -17,3 +23,104 @@ class ApplicationViewSet(
     queryset = Application.objects.filter(is_approved=True)
     lookup_field = 'client_id'
     pagination_class = None
+
+    """
+    Action view for getting the availabilty of token 
+    for a particular view in order to skip authorization
+    after first consent
+    """
+    @action(methods=['get'], detail=False, url_name="get_token", url_path="get_token")
+    def get_token(self, request):
+        
+        token=get_access_token_model().objects.filter(
+            user=request.user,
+            application=request.GET.get('id')   
+        ).exists()
+
```

Comment:
Please leave exactly one newline between blocks of code and two lines between functions and classes.

Reply:
I have made the changes. Please review